OopsConcept/inheritance_polymorphism.js

```diff
@@ -0,0 +1,24 @@
+// Inheritance example
+class person {
+	constructor(name) {
+		this.name = name;
+	}
+	// method to return the string
+	toString() {
+		return (`Name of person: ${this.name}`);
+	}
+}
+class student extends person {
+	constructor(name, id) {
+		// super keyword for calling the above
+		// class constructor
+		super(name);
+		this.id = id;
```

Comment:
What will happen, if you assign a new value to name? Which one it will take, if you print 'name' in child class?